src/containers/ResourceEmbed/components/ResourceBadge.tsx

```diff
@@ -41,6 +41,7 @@ const typeIconMapping: Record<EmbedType, ElementType> = {
   h5p: H5P,
   image: Media,
   concept: Concept,
+  gloss: Concept,
```

Comment:
Andre plasser, feks under Regler for bruk, bruker vi Globe for å skille mellom forklaring og glose. Noen spesiell grunn til at vi ikkje skal bruke det også her?

Reply:
Venter på svar fra Hedvig angående dette. Hun skulle på kontoret i dag